content/en/blog/_posts/2021-08-09-memory-manager-moves-to-beta.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: blog
+title: "Kubernetes Memory manager moves to beta"
+date: 2021-08-09
+slug: kubernetes-1-22-feature-memory-manager-moves-to-beta
+---
+
+**Authors:** Artyom Lukianov (Red Hat), Cezary Zukowski (Samsung)
+
+# Memory manager moves to beta
+
+The blog post describes the internals of the **Memory manager**, a beta feature of Kubernetes 1.22. The **Memory Manager** is a component in the **kubelet** ecosystem proposed to enable the feature of guaranteed memory (and hugepages) allocation for pods in the [Guaranteed QoS class](https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#qos-classes).
+
+This blog post covers:
+
+1. Why do you need it?
+2. The internal details of how the **MemoryManager** works
+3. Current limitations of the **MemoryManager**
+4. Future work for the **MemoryManager**
```

Comment:
Consider linking each of these to the headings within the article.

Reply:
Done!